zio-http/src/main/scala/zhttp/http/Http.scala

```diff
@@ -69,6 +71,18 @@ sealed trait Http[-R, +E, -A, +B] extends (A => ZIO[R, Option[E], B]) { self =>
   final def *>[R1 <: R, E1 >: E, A1 <: A, C1](other: Http[R1, E1, A1, C1]): Http[R1, E1, A1, C1] =
     self.zipRight(other)
 
+  /**
+   * Returns an http app that submerges the error case of an `Either` into the
+   * `Http`. The inverse operation of `Http.either`.
+   */
+  final def absolve[E1 >: E, C](implicit ev: B <:< Either[E1, C]): Http[R, E1, A, C] =
+    self.flatMap(b =>
+      ev(b) match {
+        case Right(c) => Http.succeed(c)
+        case Left(e)  => Http.fail(e)
+      },
+    )
```

Comment:
Would you mind creating a separate PR for this, so that it shows up on the release notes as a new feature & contribution :)

Reply:
Ok, I realize that you have added a ton of new operators. I guess we can write custom release notes for this PR :P